src/changes-backwards-compatible.apib

```diff
@@ -6,12 +6,13 @@ We list all backwards-compatible additions here. These are currently available i
 
 ### Latest
 
-- We added `status` on `companies.list`
-- We added `status` on `companies.info`
-- We added `status` on `contacts.list`
-- We added `status` on `contacts.info`
-- We added `ids` to the filters on `creditNotes.list`
-- We added `invoice_id` and `project_id` to the filters on `creditNotes.list`
+- We added `status` on `companies.list`.
+- We added `status` on `companies.info`.
+- We added `status` on `contacts.list`.
+- We added `status` on `contacts.info`.
+- We added `ids` to the filters on `creditNotes.list`.
+- We added `invoice_id` and `project_id` to the filters on `creditNotes.list`.
+- We added `discounts` to `creditNotes.info`.
```

Comment:
Q: Should we add this to a December header already?

Reply:
No, since December is 2 days away and the code has already been deployed

Reply:
Then a November header maybe?

Reply:
🤦  I'll dig out which ones were from November and update

Reply:
There were no updates no October, all are from November. I changed the _Latest_ segment to _November 2021_.

This seems like an easier way to work instead of doing a separate PR at the start of every month: just don't use a Latest header, always use the header of the current month. The first creator of a PR in a new month (and the reviewers) should notice that the current month is not in there yet, and thus create a new section for the current month.